scripts/generate-config.js

```diff
@@ -23,9 +23,9 @@ const splitImageFolderName = (folderName) => {
 }
 
 const getImageType = (image) => {
-  return image.name.includes('base') ? 'base' : image.name.includes('browser') ? 'browser' : 'included'
+  return image.includes('base') ? 'base' : image.includes('browser') ? 'browser' : 'included'
```

Comment:
```suggestion
  return imageType.includes('base') ? 'base' : imageType.includes('browser') ? 'browser' : 'included'
```

I think it's better to keep this name in parity with what you have on line 28.